### PR TITLE
Add UI and logic Sign Up screen

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Code UI and Logic for Sign up Screen

## Task
Link of task: 
1. https://trello.com/c/brJcm61P
2. https://trello.com/c/UMYqKfHy

## Screenshot
image

Screenshot_1696320667
Screenshot_1696320673

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/68725f86-c7ad-445e-942c-3aeb970da84f

https://github.com/dustin-goldenowl/jobspot/assets/144195318/b0114322-2052-4ccf-aa3b-b7df6fdce605

### CODE REVIEW

File: lib/src/presentations/sign_up/screen/sign_up_view.dart [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+class SignUpView extends StatefulWidget {
+  const SignUpView({super.key});
+
+  @override
+  State<SignUpView> createState() => _SignUpViewState();
+}
+
+class _SignUpViewState extends State<SignUpView>
+    with SingleTickerProviderStateMixin {
+  late TabController tabController;
```
Comment (dustin-goldenowl) [inline lib/src/presentations/sign_up/screen/sign_up_view.dart]: put "TabController tabController" in SignUpState

File: lib/src/presentations/sign_up/widgets/birthday_widget.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  });
+
+  final String title;
+  final Function(DateTime date) onChange;
+  final DateTime selectedDate;
+  final DateTime lastDate;
+
+  @override
+  Widget build(BuildContext context) {
+    final formater = DateFormat('dd/MM/yyyy');
```
Comment (dustin-goldenowl) [inline lib/src/presentations/sign_up/widgets/birthday_widget.dart]: Should be defin a class about format date time. Example : DateTimeUtils ...

File: lib/src/presentations/sign_up/widgets/gender_widget.dart [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+  final bool isMale;
+  final Function(bool value) onChange;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text("Gender", style: AppStyles.boldTextNightBlue),
```
Comment (dustin-goldenowl) [inline lib/src/presentations/sign_up/widgets/gender_widget.dart]: Put in app_en.arb

File: lib/src/presentations/sign_up/widgets/gender_widget.dart [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        const SizedBox(height: 10),
+        Row(
+          children: [
+            Expanded(
+              child: _buildGender(
+                title: "Male",
```
Comment (dustin-goldenowl) [inline lib/src/presentations/sign_up/widgets/gender_widget.dart]: The same above

File: lib/src/presentations/sign_up/widgets/birthday_widget.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/sign_up/widgets/birthday_widget.dart]: I fixed it

File: lib/src/presentations/sign_up/widgets/gender_widget.dart (hunk 3 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/sign_up/widgets/gender_widget.dart]: I fixed it

File: lib/src/presentations/sign_up/widgets/gender_widget.dart (hunk 4 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/sign_up/widgets/gender_widget.dart]: I fixed it

File: lib/src/presentations/sign_up/screen/sign_up_view.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/sign_up/screen/sign_up_view.dart]: I fixed it

File: lib/src/core/utils/date_time_utils.dart [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import 'package:intl/intl.dart';
+
+class DateTimeUtils {
+  static DateFormat formatDMY = DateFormat('dd/MM/yyyy');
```
Comment (dustin-goldenowl) [inline lib/src/core/utils/date_time_utils.dart]: In this Class, Can you create 1 function return dateTime has formatted with input DateTime , output String

Comment (WalterGoldenOwl) [reply lib/src/core/utils/date_time_utils.dart]: I fixed it

**Pull request merged**